### DISCUSSION TITLE
Particle Tracking with ray tracing

### DISCUSSION THREAD

**Discussion by gsgall** [idaholab/moose#25376] [Q&A Modules: General] [answered]
I am starting to work on an Application that plans to use the Raytracing module to track time dependant particle motion. I have seen a few discussions about this on the forum but I am still not sure how to do this. Are there any good examples available that show how to track the motion of say a single particle as a function of time using the Ray Tracing module?

**Comment by GiudGiud** [comment, marked as answer]
Hello

There's this branch that was started to do this for the material point method:
https://github.com/loganharbour/moose/tree/rt_mpm
Then there is MaCaw a ray-tracing openmc-based monte carlo code we're trying to open source. I ll post a link when it's released

We will have work starting in October on particle-in-cell methods that will also do something similar 

Guillaume
